src/main/webapp/feed.html

```diff
@@ -9,6 +9,18 @@
     <script src="/js/feed-loader.js"></script>
 </head>
 <body onload="buildUI()">
+
+    <nav>
+    <div class="dropdown"style="float:right;">
+      <button class="dropbtn">Menu</button>
+      <div class="dropdown-content">
+      <ul id="navigation">
+        <li><a href="/">Home</a></li>
+        <li><a href="/stats.html">Stats</a></li>
```

Comment:
Do we also want them to be able to navigate to their user page while looking at the feed?

Reply:
Still trying to get it so that users can navigate to their page. Worst case, they'll have to go back to the home page before going back to their page. Re-routing this as I do in the index.html file doesn't work as easily, unfortunately.